packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -67,12 +72,20 @@ class ScaleUpdateDetails {
   /// greater than or equal to zero.
   final double scale;
 
+  /// The scale along the horizontal axis implied by the pointers in contact
+  /// with the screen. A value greater than or equal to zero.
```

Comment:
Please add a "See also:" section to the docs of scale, horizontalScale, and verticalScale linking to each other and explaining the difference between the three.